src/exodus/exoduswallet/test/wallet_tests.cpp

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2019 The Zcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <boost/test/unit_test.hpp>
+
+#include "wallet/wallet.h"
+#include "wallet/test/wallet_test_fixture.h"
+
+#include "../wallet.h"
+
+struct ExodusSigmaTestingSetup : WalletTestingSetup
+{
+    ExodusSigmaTestingSetup()
+        : WalletTestingSetup()
+    {
+        exodusTestWallet = new exodus::ExodusWallet(pwalletMain->strWalletFile);
+    }
+
+    ~ExodusSigmaTestingSetup()
+    {
+        delete exodusTestWallet;
+        exodusTestWallet = nullptr;
+    }
+    exodus::ExodusWallet *exodusTestWallet;
```

Comment:
We can make it as non-pointer so it does not need to delete manually.

Reply:
we need to define default constructor for `ExodusWallet`. I think this is fine.

Reply:
We can invoke another constructor instead of default one for member. See here: https://en.cppreference.com/w/cpp/language/initializer_list